### PR TITLE
[OPS-7438] Edgierlord - PHP8!

### PR DESCRIPTION
**Pull request by cafuego**

### CODE REVIEW

File: alpine-base-php/php8/etc/php8/conf.d/01_wddx.ini [hunk 1]
```diff
@@ -0,0 +1 @@
+extension=wddx.so
```
Comment (orakili) [inline alpine-base-php/php8/etc/php8/conf.d/01_wddx.ini]: Looks like this extension has been deprecated: https://www.php.net/manual/en/intro.wddx.php and it's not present anymore:

> PHP Warning:  PHP Startup: Unable to load dynamic library 'wddx.so' (tried: /usr/lib/php8/modules/wddx.so (Error loading shared library /usr/lib/php8/modules/wddx.so: No such file or directory), /usr/lib/php8/modules/wddx.so.so (Error loading shared library /usr/lib/php8/modules/wddx.so.so: No such file or directory)) in Unknown on line 0

File: alpine-base-php/php8/etc/php8/conf.d/00_json.ini [hunk 2]
```diff
@@ -0,0 +1 @@
+extension=json.so
```
Comment (orakili) [inline alpine-base-php/php8/etc/php8/conf.d/00_json.ini]: This is now always included in PHP: https://php.watch/versions/8.0/ext-json

For info:

> PHP Warning:  PHP Startup: Unable to load dynamic library 'json.so' (tried: /usr/lib/php8/modules/json.so (Error loading shared library /usr/lib/php8/modules/json.so: No such file or directory), /usr/lib/php8/modules/json.so.so (Error loading shared library /usr/lib/php8/modules/json.so.so: No such file or directory)) in Unknown on line 0

**Review by orakili — COMMENTED** [review]
Tested the php8 builder and k8s images for RW9 and they are working well. Only some warnings for the `wddx` and `json` extentsions. The former because it was deprecated, the latter because it's included in PHP 8 now.

File: alpine-base-php/php8/etc/php8/conf.d/01_wddx.ini (hunk 1 above)
Comment (cafuego) [reply alpine-base-php/php8/etc/php8/conf.d/01_wddx.ini]: Ah, yes. I had removed the package from the Dockerfile but forgot to remove the config snippet which is separate in the repository.

**Pull request merged**